go.mod

```diff
@@ -5,5 +5,6 @@ go 1.18
 require (
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.1.0-rc2
+	golang.org/x/exp v0.0.0-20221114191408-850992195362
```

Comment:
Agree. It is not worthy to add dependencies just for `slices.Clone`, which is easy to implement.
```go
func Clone[S ~[]E, E any](s S) S {
	// Preserve nil in case it matters.
	if s == nil {
		return nil
	}
	return append(S([]E{}), s...)
}
```

Reply:
Done.